client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/Forms/ExposureForm/ExposureForm.tsx

```diff
@@ -51,24 +52,21 @@ const ExposureForm = (props: Props) => {
 		exposureAndFlightsData,
 		exposureSourceSearchString,
 	});
-	useEffect(() => {
-		const setOptionalCovidPatientsAsync = async () => {
-			const optionalCovidPatients = await fetchOptionalCovidPatients({ setIsLoading });
-			setOptionalCovidPatients(optionalCovidPatients);
-		};
-		setOptionalCovidPatientsAsync();
-	}, [exposureSourceSearchString]);
+	const setOptionalCovidPatientsAsync = async () => {
+		const optionalCovidPatients = await fetchOptionalCovidPatients({ setIsLoading });
```

Comment:
what is the usement for setIsLoading

Reply:
changed it to the prop of useExposureForm instead of in the function [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1025/commits/960f81b6b0e40047540ec331e23f046c993a58cb)

Reply:
just asking what is `[isLoading, setIsLoading]` state stands for?

Reply:
showing the loading icon (on the list itself)  while the list of optional covid patients is loading , i've renamed it to isOptionalPatientsLoading instead [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1025/commits/960f81b6b0e40047540ec331e23f046c993a58cb)